exercises/practice/rational-numbers/.docs/instructions.md

```diff
@@ -28,16 +28,15 @@ Exponentiation of a real number `x` to a rational number `r = a/b` is `x^(a/b) =
 
 Implement the following operations:
 
-- conversion and promotion of integers to rationals. Read the manual's chaper on [conversion and promotion](https://docs.julialang.org/en/v1/manual/conversion-and-promotion/) for details about how and why to do this.
-- addition, subtraction, multiplication and division of two rational numbers
-- absolute value, exponentiation of a given rational number to an integer power, exponentiation of a real number to a rational number
-- comparison (less than, equals, etc.) of two rational numbers, comparison of a rational number and an integer
-
-In Julia, there are fallback methods already defined for many arithmetic methods, comparisons, etc, especially for subtypes of `Real`.
-You do not need to write your own method for `>=` if you have already defined `<`; nor do you need to define `^(::RationalNumber, ::Integer)` if you have provided a method for multiplication; etc.
-You can examine the fallback methods for a given function by entering `methods(abs)` or `edit(abs, (RationalNumber, RationalNumber))` at the REPL.
-
-Your implementation of rational numbers should always be reduced to lowest terms. For example, `4/4` should reduce to `1/1`, `30/60` should reduce to `1/2`, `12/8` should reduce to `3/2`, etc. To reduce a rational number `r = a/b`, divide `a` and `b` by the greatest common divisor (gcd) of `a` and `b`. So, for example, `gcd(12, 8) = 4`, so `r = 12/8` can be reduced to `(12/4)/(8/4) = 3/2`.
-The reduced form of a rational number should be in "standard form" (the denominator should always be a positive integer). If a denominator with a negative integer is present, multiply both numerator and denominator by `-1` to ensure standard form is reached. For example, `3/-4` should be reduced to `-3/4`
```

Comment:
There's Julia-specfic stuff in here that should be retained. Without these hints the exercise is harder and students will probably learn less.

Reply:
The first set of bullet points also contains content that should be retained:

```
- conversion and promotion of integers to rationals. Read the manual's chaper on [conversion and promotion](https://docs.julialang.org/en/v1/manual/conversion-and-promotion/) for details about how and why to do this.
- comparison (less than, equals, etc.) of two rational numbers, comparison of a rational number and an integer
```

IMO, it is important to keep hinting to the student that conversion and promotion is the first thing they should do. If you want to move this into the .append file then please find some way of making it more prominent.

Otherwise, maybe just revert the changes to the files in this directory.

Reply:
I'd rather move the Julia-specific information to an append as long as we're not duplicating information in the instructions. That's the suggested workflow in the Exercism docs and I don't see a good reason to deviate. Editing the instructions in place just delays the issue.

Reply:
I appreciate your work in revising this PR. Are you saying that you intend to move those two lines into a .append file?

As I say below, I think that's fine so long as the advice to start with conversion and promotion is made prominent (ideally even more prominent than it is now).

Re: reverting, you don't see a good reason to deviate from the suggested workflow.

The rational-numbers exercise, like several others on the Julia track, has a bunch of intentional differences from the upstream definitions in problem-specs. I think the julia-specific changes are good and that if we remove them it should be because they are bad for students or mentors, not because they are inconsistent with problem-specifications.

What is the cost of letting rational-numbers continue to differ from the problem-spec version? I don't work on this repo very much any more, is there a developer experience issue that I'm not seeing?